Sources/ApolloCodegenLib/FileGenerators/SchemaFileGenerator.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import OrderedCollections
+
+/// Generates a file containing schema metadata used when converting data to `Object` types at runtime.
+struct SchemaFileGenerator: FileGenerator, Equatable {
+  /// The schema module name.
+  let name: String
+  /// The `OrderedSet` of `GraphQLObjectType` objects used to build the file content.
+  let objectTypes: OrderedSet<GraphQLObjectType>
+  let path: String
+
+  var data: Data? {
+    #warning("TODO: need correct data template")
+    return "public enum Schema {}".data(using: .utf8)
```

Comment:
`SchemaTemplate` already exists. This should just be `SchemaTemplate(schema: schema).render()`.

Reply:
This is addressed in #2097